index.js

```diff
@@ -1,95 +1,129 @@
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
+const name = "Victor"
 
-// console.log(name)
+console.log(name)
 // // => "Ian"
 // name = "Not Ian" 
 // // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+function drinkWater(x){
+  console.log("Man I sure am thirsty")
+  x--
+  console.log("Ahh that hits the spot")
+  return x
+}
 
-// console.log(drinkWater(12))
+console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum){
+  if(num === maybeNum){
+    return "same same"
+  } else if(num == maybeNum){
+    return "same same (but different)"
+  } else {
+    return "different"
+  }
+}
 
-// console.log(sameSameButDifferent(5, 5)) 
+console.log(sameSameButDifferent(5, 5)) 
 // // => "same same"
 //
-// console.log(sameSameButDifferent(123, "123")) 
+console.log(sameSameButDifferent(123, "123")) 
 // // => "same same (but different)"
 //
-// console.log(sameSameButDifferent(5, 7)) 
+console.log(sameSameButDifferent(5, 7)) 
 // // => "different"
 //
-// console.log(sameSameButDifferent(123, "122")) 
+console.log(sameSameButDifferent(123, "122")) 
 // // => "different"
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
+console.log("%cQuestion 5", "color: red")
 
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)  
+function updateGrade(student, grade){
+  student.grade = grade
+}
+
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+console.log(student1)  
 // // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 6 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
+console.log("%cQuestion 6", "color: red")
+
+function printNameAndPhones(users){
+  users.forEach(user => {
+    console.log(user.name)
+    console.log(`Cell: ${user.phones["cell"]}`)
+    console.log(`Office: ${user.phones["office"]}`)
+  });
```

Comment:
Love to see the arrow function syntax! We'll go over that a bit later this week, but arrow functions are really nice for callbacks.

For accessing the user's phones, you can also use dot notation (since `.phones` is also an object):

```js
    console.log(`Cell: ${user.phones.cell}`)
    console.log(`Office: ${user.phones.office}`)
```